components/RemoveFromCart.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import cookie from 'js-cookie';
+
+type Item = {
+  id: string;
+  name: string;
+  img: string;
+  amount: number;
+  price: number;
+  info: string;
+};
+
+type itemsInCart = {
+  id: string;
+  name: string;
+  img: string;
+  amount: number;
+  price: number;
+  info: string;
+};
+type Props = { item: Item; itemsInCart: itemsInCart[] };
+
+export default function RemoveFromCart(props: Props) {
+  function removeItem(id) {
```

Comment:
You may want to move these "cookies" functions all out into a separate file (`cookies.ts`?) so that you can have all cookie logic in one place (like `db.js`)